test/pkg/environment/common/expectations.go

```diff
@@ -159,6 +203,15 @@ func (env *Environment) EventuallyExpectHealthy(pods ...*v1.Pod) {
 	}
 }
 
+func (env *Environment) EventuallyExpectKarpenterRestarted() {
+	env.EventuallyExpectKarpenterRestartedWithOffset(1)
+}
+
+func (env *Environment) EventuallyExpectKarpenterRestartedWithOffset(offset int) {
```

Comment:
We don't need this with GinkgoHelper() right? If you're opting to not migrate it all in this PR, fine with me too :)